styleguide/components/71-pages/general/info-page/info-page--variant1.twig

```diff
@@ -22,37 +22,37 @@
     {% include '@image-gallery--multiple' with {
     "items": [
     {
-    "src": "https://via.placeholder.com/700x432",
+    "src": "https://via.placeholder.com/735x459",
```

Comment:
Gebruiken we hier en in de andere pages niet beter echte images? Deze worden namelijk gebruikt ter validatie vanaf volgende week.

Reply:
Updated those in the extra articles block.